.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
```

Comment:
I think the `github-release` Action should trigger this one, because that creates a Github release.